tests/integration/update_cluster/externallb/kubernetes.tf

```diff
@@ -86,7 +86,7 @@ resource "aws_autoscaling_group" "master-us-test-1a-masters-externallb-example-c
     id      = aws_launch_template.master-us-test-1a-masters-externallb-example-com.id
     version = aws_launch_template.master-us-test-1a-masters-externallb-example-com.latest_version
   }
-  load_balancers      = ["my-external-elb-2", "my-external-elb-3"]
+  load_balancers      = ["master-us-test-1a.masters.externallb.example.com-my-external-elb-1", "master-us-test-1a.masters.externallb.example.com-my-external-elb-2", "master-us-test-1a.masters.externallb.example.com-my-external-elb-3"]
```

Comment:
I wouldn't expect the terraform and cloudformation outputs to be changing. We don't want to change the names of the load balancers we're attaching to the ASGs, only the names of their fi tasks

Reply:
True. These latest changes should do the trick.
I guess now this has to its way into 1.19 and 1.18?